src/Components/configuration-panel/index.js

```diff
@@ -0,0 +1,14 @@
+import './index.css'
```

Comment:
Do you mind if we rename component files with their name instead of index?
e.g. configuration-panel.js, or configuration-panel.css

Reply:
Ah nvm, i see "dropdown" isn't defined in the value selectors [here](https://github.com/nickywongdong/timr/blob/7f7c718e2829c0a463093e5a565493828ea051b2/src/Components/value-selector/index.js)

Reply:
actually i'd like to follow this rulebook:
https://github.com/airbnb/javascript/tree/master/react
And it seems to want to follow this pattern.
Instead we should have component names be PascalCase instead, so ConfigurationPanel isntead of configuration-panel

Reply:
ok ill read through this